apis/infix-en_GB/atrium-api-infix-en_GB-jvm/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
         exclude module: "${rootProject.name}-translations-en_GB-jvm"
     }
     testImplementation prefixedProject('translations-de_CH-jvm')
+    api niok()
```

Comment:
do we need this?

Reply:
I'll try building without it and see if it works

Reply:
if we do, then please move up to other api dependencies